README.markdown

```diff
@@ -22,6 +22,7 @@ The build install instructions are currently being written. The Xapi
 Project contains a large list of dependencies and sub-projects, which
 are actually quite difficult to build independently. To build xen-api
 from source, we recommend using [dune (previously named jbuilder)](https://github.com/ocaml/dune)
+and [opam](https://opam.ocaml.org/doc/Manual.html) using the [xs-opam](https://github.com/xapi-project/xs-opam) remote.
```

Comment:
I'd completely get rid of the dune line, it is a necessary requirement not a suggestion. Just write

> To build xen-api from source, we recommend using [opam](https://opam.ocaml.org/doc/Manual.html) with the [xs-opam](https://github.com/xapi-project/xs-opam) remote, refer to its [README](https://github.com/xapi-project/xs-opam/blob/master/README.md#adding-the-opam-repository) for a more detailed explanation.